matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/TokenChunkEventPersistor.kt

```diff
@@ -67,14 +67,25 @@ internal class TokenChunkEventPersistor @Inject constructor(
                 .awaitTransaction { realm ->
                     Timber.v("Start persisting ${receivedChunk.events.size} events in $roomId towards $direction")
 
+                    /**
+                     * As per spec
+                     * if no further events are available (either because we have reached the start of the timeline,
+                     * or because the user does not have permission to see any more events), the end property is omitted from the response.
+                     * Synapse is not omitting it and EA was relying on it
+                     */
+                    val chunkEnd = if (receivedChunk.events.isEmpty()) {
+                        receivedChunk.start
```

Comment:
ReceivedChunk.start should be the same as the from param, so we will be blocked too, no?

Reply:
I think the 'start' is set but 'end' is unset (=null?). Probably chunkEnd=null causes these problems in Element Android